examples/milvus/locustfile.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env python3
```

Comment:
Can this example be simplified? Does introducing numpy as a dependency really make sense?

No need to print instructions manually on execution of __main__, rely on the arg help text or comments in code.

Reply:
Great. Go ahead and remove the numpy stuff too. I like keeping examples as minimalist as possible.